docs/test-readmodels.md

```diff
@@ -0,0 +1,92 @@
+---
+id: test-readmodels
+title: Test Read Models
+---
+
+## Testing Tools
+
+The **resolve-testing-tools** package contains utilities that allow you to test Read Models and Sagas. The provided tools reproduce a Read Model's full lifecycle in the testing environment.
+
+To write a test, call the `givenEvents` function. This function takes an array of events and gives access to a number of functions that you can chain together to describe the test case. The following functions are available:
```

Comment:
What do you mean when you say "gives access to functions"?

Reply:
It returns an object that contains these functions. The idea is that you can access these functions through the `givenEvents` function.